src/components/Description/index.js

```diff
@@ -1,7 +1,15 @@
 import { StyledDescription } from "./description.styles";
 
 const Description = ({ description }) => {
-  return <StyledDescription>{description}</StyledDescription>;
+  const htmlDescription = () => {
+    return { __html: `${description}` };
+  };
+
+  return (
+    <StyledDescription
+      dangerouslySetInnerHTML={htmlDescription()}
+    ></StyledDescription>
```

Comment:
is it necessary to create that function? 

`<StyledDescription   dangerouslySetInnerHTML={{ __html: description }} />`